src/test/scala/com/github/saurfang/sas/SasRelationSpec.scala

```diff
@@ -3,28 +3,24 @@ package com.github.saurfang.sas
 import com.ggasoftware.parso.SasFileConstants
 import org.apache.hadoop.mapred.JobConf
 import org.apache.spark.sql.{SQLContext, SaveMode}
-import org.apache.spark.{Logging, SharedSparkContext}
+import org.apache.spark.{SharedSparkContext}
 import org.scalactic.TolerantNumerics
 import org.scalatest._
+import org.apache.log4j.LogManager
+import org.apache.spark.sql.SparkSession
 
-class SasRelationSpec extends FlatSpec with Matchers with Logging with SharedSparkContext{
+class SasRelationSpec extends FlatSpec with Matchers with SharedSparkContext{
   val BLOCK_SIZE = 3 * 1024 * 1024
 
   "SASReltion" should "read SAS data exactly correct" in {
-    val sqlContext = new SQLContext(sc)
+    val spark = SparkSession.builder().getOrCreate()
 
-    val jobConf = new JobConf()
```

Comment:
If you are not passing the following settings to the test, it will not actually split the SAS file and unable to verify the correctness of parallelism.